config/sets/symfony/symfony7/symfony73.php

```diff
@@ -8,5 +8,6 @@
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->import(__DIR__ . '/symfony73/symfony73-console.php');
+    $rectorConfig->import(__DIR__ . '/symfony73/symfony73-security.php');
```

Comment:
should be equal with symfony component-name:

```suggestion
    $rectorConfig->import(__DIR__ . '/symfony73/symfony73-security-core.php');
```

Reply:
Yes, fixed here and below